content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -124,6 +124,7 @@ When reviewing, use the following as a starting point.
 ### Language and grammar
 
 - Are there any obvious errors in language or grammar? Is there a better way to phrase something?
+  - You should focus on the language and grammar of the parts of the page that is being changed. Unless the author is clearly aiming to update the whole of an existing page, it's OK for them not to fix everything that could be improved in it.
```

Comment:
Maybe consider adding a note of what to do if you spot additional changes outside of the scope of the PR, then file an issue of open a new PR.